### PR TITLE
Added plugin information in getting started doc

### PR DESCRIPTION
**Pull request by debasishg**
### What changes were proposed in this pull request?

* Documentation change - add `cloudflow-plugin` information in Getting Started document

### Why are the changes needed?

* This is a critical information for developers - hence required to be included

### Does this PR introduce any user-facing change?

* Yes. Documentation change

### How was this patch tested?

* Nothing to test - just add documentation

### CODE REVIEW

File: docs/Getting Started.md [hunk 1]
```diff
@@ -90,7 +92,17 @@ Cloudflow offers several sbt plugins that abstract quite a bit of boilerplates n
...
 
-The above build script is standard Scala sbt - the only difference is the plugin which we provide as part of Cloudflow.
+The above build script is standard Scala sbt - the only difference is the plugin which we provide as part of Cloudflow. And here's how we specify the name and version of the plugin to use:
+
+**project/cloudflow-plugins.sbt**
+
+```
+// Resolver for the cloudflow-sbt plugin
+//
+resolvers += "Akka Snapshots" at "https://repo.akka.io/snapshots/"
```
Comment (agemooij) [inline docs/Getting Started.md]: @debasishg this resolver is not required for resolving the sbt plugin. AFAIK this esolver is not required for any part of Cloudflow and should be removed where we still see it.

Background: a long time ago we added some features to Akka Streams and while those features were not released yet we used Akka snapshots.

File: docs/Getting Started.md [hunk 2]
```diff
@@ -90,7 +92,17 @@ Cloudflow offers several sbt plugins that abstract quite a bit of boilerplates n
...
+
+addSbtPlugin("com.lightbend.cloudflow" % "sbt-cloudflow" % "1.3.0-M1")
```
Comment (agemooij) [inline docs/Getting Started.md]: Shouldn't this be M5?

File: docs/Getting Started.md (hunk 1 above)
Comment (debasishg) [reply docs/Getting Started.md]: ok - will remove it ..

File: docs/Getting Started.md (hunk 2 above)
Comment (debasishg) [reply docs/Getting Started.md]: ok - will change .. finally we should have `1.3.0`

**Comment by debasishg** [comment]
@agemooij Incorporated feedbacks ..

**Pull request merged**